.github/workflows/testing.yml

```diff
@@ -33,43 +31,39 @@
            - environment-file: .ci/311.yaml
              os: windows-latest
        fail-fast: false
-    
+
+     defaults:
+       run:
+         shell: bash -l {0}
+
      steps:
        - name: checkout repo
          uses: actions/checkout@v3
        
        - name: setup micromamba
-         uses: mamba-org/provision-with-micromamba@main
+         uses: mamba-org/setup-micromamba@v1
          with:
            environment-file: ${{ matrix.environment-file }}
            micromamba-version: 'latest'
-           channel-priority: 'flexible'
         
        ##############################################################################
        #- name: install pulp via pip
-       #  shell: bash -l {0}
-       #  run: pip install pulp
+       #  run: |
```

Comment:
What is this?

Reply:
This something that I want to keep in for now in case we need to uncomment in the future. Basically, `pulp`'s solver installation is a bit varied by OS and installation method. Sometimes previously we need to run `pulptest` to get the proper solvers installed and available following actual `pulp` installation. xref #348